packages/driver/cypress/e2e/commands/assertions.cy.js

```diff
@@ -790,19 +793,18 @@ describe('src/cy/commands/assertions', () => {
       cy.noop({}).should('have.property', 'foo')
     })
 
-    it('ends and snapshots immediately and sets child', (done) => {
+    it('snapshots immediately and sets child', (done) => {
       cy.on('log:added', (attrs, log) => {
-        if (attrs.name === 'assert') {
-          cy.removeAllListeners('log:added')
+        if (attrs.name !== 'assert') {
+          return
+        }
 
-          expect(log.get('ended')).to.be.true
-          expect(log.get('state')).to.eq('passed')
```

Comment:
Are theses no longer true?

Reply:
Correct. I put an explanation of the change here: https://github.com/cypress-io/cypress/pull/23354#discussion_r946210351